parsers/AR.py

```diff
@@ -143,8 +143,44 @@ def fetch_exchange(
     logger: Logger = getLogger(__name__),
 ) -> dict:
     """Requests the last known power exchange (in MW) between two zones."""
+    if target_datetime:
+        raise NotImplementedError("This parser is not yet able to parse past dates")
+
+    sorted_zone_keys = sorted([zone_key1, zone_key2])
+    sorted_codes = "->".join(sorted_zone_keys)
+    flow = 0
+    target_datetime = ""
+
+    if sorted_codes in SUPPORTED_EXCHANGES:
+      current_session = session or Session()
+
+      api_cammesa_response = current_session.get(CAMMESA_EXCHANGE_ENDPOINT)
+      assert api_cammesa_response.status_code == 200, (
+        "Exception when fetching echange for "
+        "{}: error when calling url={}".format(
+          api_cammesa_response.status_code, CAMMESA_EXCHANGE_ENDPOINT
+        )
+      )
+
+      exchange_name = SUPPORTED_EXCHANGES[sorted_codes]
+      exchange_list = api_cammesa_response.json()
+      for exchange in exchange_list["features"]:
+        properties = exchange["properties"]
+        if properties["nombre"] == exchange_name:
+          flow = -int(properties["text"])
```

Comment:
Noob question: why do we want the negative of the value?

Reply:
b/c we write the exchanges as `[country A]->[country B]`, where A is lexicographically before B.

- A negative value indicates that A is **importing** from B
- A positive value indicates that A is **exporting** to B.

Reply:
With the change I made, we're actually doing the opposite! A negative value is export (A->B) and positive is import (B->A). I did this because I saw that that was how AR->BR_S was doing it! It doesn't really matter, as long as the arrow angle is correct 😊